tests/smartswitch/platform_tests/test_reload_dpu.py

```diff
@@ -180,8 +202,22 @@ def test_dpu_status_post_dpu_kernel_panic(duthosts, dpuhosts,
     logging.info("Checking DPUs are not pingable")
     check_dpus_are_not_pingable(duthost, ip_address_list)
 
+    logging.info("Checking DPUs reboot reason as Kernel Panic")
+    check_dpus_reboot_cause(duthost, dpu_on_list, num_dpu_modules, "Kernel Panic")
+
+    logging.info("Shutdown DPUs after kernel Panic")
+    dpus_shutdown_and_check(duthost, dpu_on_list, num_dpu_modules)
```

Comment:
Why the shutdown is required? The DPU should be rebooted automatically by the watchdog.
Please add a condition if this is vendor specific behavior.